code/src/index.css

```diff
@@ -1,13 +1,320 @@
+		sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	
+	font-weight: 500;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+		monospace;
 }
+
+.container {
+	padding: 15px 25px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background-color: rgb(255, 255, 255);
+	
+	
+}
+
+.ContainerData{
+width: 100%;
+display: flex;
+align-items: center;
+justify-content: center;
+flex-direction: column;
+
+}
+
+
+/* Container over post and masage */
+
+
+.messageContainer {
+	display: flex;
+	align-items: center;
+	justify-items: center;
+	flex-direction: column;
+width: 100%;
+	
+	
+	
+}
+
+
+
+
+
+/* Make a post area */
+
+
+
+
+
+
+
+
+
+.contentTextArea {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+	margin-bottom: 25px;
+	background-color: rgb(242, 240, 240);
+	border: rgb(0, 0, 0) 1px solid;
+	border-radius: 1%;
+	box-shadow: 12px 12px 1px -5px black;
+	width: 100%;
+	max-width: 450px;
+	height: 140px;
+	white-space: normal;
+	word-break: normal;
+	padding: 10px 15px;
+	font-size: 15px;
+	font-weight: 700;
+}
+
+.TextArea {
+	display: flex;
+	justify-content: center;
+	height: 40px;
+	margin: 5px 0px 15px 0px;
+	
+	
+
+	
+}
+textarea {
+	background-color: rgb(255, 255, 255);
+	width: 100%;
+	height: 90%;
+	border: lightgray 1.5px solid;
+	resize: none;
+	overflow: hidden;
+    overflow-wrap: break-word;
+padding: 7px 0 7px 7px;
+font-family: monospace;
+font-weight: 600;
+font-size: 13px;
+letter-spacing: 2px;
+
+
+
+
+}
+textarea:focus {
+	border: none;
+	border: rgb(185, 184, 184) 2px solid;
+	outline: none;
+	
+  }
+
+  textarea:focus:invalid {
+	outline: none;
+	border: rgb(185, 184, 184) 2px solid;
+	background: url("https://assets.digitalocean.com/labs/icons/exclamation-triangle-fill.svg") no-repeat 95% 50% rgb(255, 255, 255);
+	background-size: 25px;
+	
+  }
+
+.counterButton{
+
+display: flex;
+justify-content: space-between;
+align-items: center;
+
+}
+
+.counterText{
+	font-size: 12px;
+	color: rgb(170, 168, 168);
+
+
+}
+
+
+
+.PostBtn {
+	background-color: rgb(255, 173, 173);
+	border: none;
+	border-radius: 100px;
+	padding: 9px 15px;
+	font-size: 13px;
+	font-weight: 600;
+	cursor: pointer;
+	margin: 10px 0;
+
+}
+.PostBtn:hover {
+	background-color: rgb(245, 150, 150);
+}
+
+
+
+
+
+
+/* Message container  */
+
+
+.content {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+
+	margin-bottom: 25px;
+	background-color: rgb(242, 240, 240);
+	border: rgb(0, 0, 0) 1px solid;
+	border-radius: 1%;
+	box-shadow: 12px 12px 1px -5px black;
+	width: 100%;
+	max-width: 450px;
+	height: 140px;
+	white-space: normal;
+	word-break: normal;
+	padding: 10px 15px;
+	font-size: 15px;
+	font-weight: 700;
+}
+
+/* Text-box message */
+
+
+.TextPost{
```

Comment:
It is not necessary to name classes with uppercase, I believe only components should be named with uppercase.

Reply:
Thanks for a superb feedback 🦾 that help me improve ! And Im going to implement name correct(I totally missed out on how to do it right) and be better to comment my code! I have a problem with prittier that im trying to fix(oh my code spacing funky!!) 

Thanks :D